src/content/docs/infrastructure/elastic-container-service-integration/installation/install-ecs-integration.mdx

```diff
@@ -16,9 +16,9 @@ New Relic's ECS integration reports and displays performance data from your [Ama
 
 Before you install our ECS integration, we recommend reviewing the [requirements](/docs/introduction-amazon-ecs-integration#requirements). During the install process:
 
-* **For EC2 and EXTERNAL launch type:** The infrastructure agent (`newrelic-infra`) gets deployed onto an ECS cluster as a service using the [daemon scheduling](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/scheduling_tasks.html) strategy. This deployment installs the infrastructure agent in all the container instances of the cluster. The infrastructure agent then monitors ECS and Docker containers.
+* **For EC2 and ECS Anywhere launch type:** The infrastructure agent (`newrelic-infra`) gets deployed onto an ECS cluster as a service using the [daemon scheduling](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/scheduling_tasks.html) strategy. This deployment installs the infrastructure agent in all the container instances of the cluster. The infrastructure agent then monitors ECS and Docker containers.
```

Comment:
I think this could be confusing for users since `ECS Anywhere` is not the launch type that AWS uses. I would keep `External` which is the one used by AWS. 
```suggestion
* **For EC2 and External (ECS Anywhere) launch type:** The infrastructure agent (`newrelic-infra`) gets deployed onto an ECS cluster as a service using the [daemon scheduling](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/scheduling_tasks.html) strategy. This deployment installs the infrastructure agent in all the container instances of the cluster. The infrastructure agent then monitors ECS and Docker containers.
```